plugins/services/src/js/containers/services/ServicesTable.js

```diff
@@ -391,6 +392,23 @@ class ServicesTable extends React.Component {
     );
   }
 
+  renderInstances(prop, service) {
+    const isFolder = service.isFolder();
+    const instancesCount = service.getInstancesCount();
+    const tasksRunning = service.getTaskCount();
+    const overview = tasksRunning === instancesCount
+      ? ` ${tasksRunning}`
+      : ` ${tasksRunning}/${instancesCount}`;
+
+    const content = instancesCount == null || isFolder ? "\u2014" : overview;
```

Comment:
Wonder why you assign the return  value of `isFolder` in line 396 instead of simply calling it in the expression?

Reply:
I found 1% cleaner to store in a variable having at the top the method and then using the variable. Nothing special here.